tgui/packages/tgui/interfaces/DonorGear.js

```diff
@@ -0,0 +1,39 @@
+import { useBackend } from '../backend';
```

Comment:
I do not know if js files can be autodoc'd, but assuming they can
```suggestion
/**
  * [A short overview of how this works]
  *
  * A longer
  * paragraph of functionality about the class
  * including any assumptions/special cases
  *
  */

import { useBackend } from '../backend';
```
I would check with Mr. Kar to see.